Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -384,12 +412,20 @@ void ImageEditor::keydown_event(GUI::KeyEvent& event)
         return;
     }
 
+    if (event.key() == Key_Alt) {
+        m_select_color_mode = true;
+    }
```

Comment:
I think you don't need to track alt manually but can just use `MouseEvent::alt()` in `mousedown_event`.